### PR TITLE
Improve static files cdn

### PR DESCRIPTION
**Pull request by FabriceIRANKUNDA**
## Summary
_A summary of changes being made in this PR_
Able:
to list all files and folders of the project in the admin system
Navigate through folders
Preview all files in both editor and admin system
Rename files and folders
View properties of a file and folder (size, name, type and URL) in both editor and admin system

## Checklist
- [x] Pre-push checks pass `npm run check`
  - [x] Linter passing via `npm run lint`
  - [x] Unit & Integration tests passing via `npm run test:packages`
  - [x] Docker build process passing via `npm run build-client`
- [x] If this PR is still a WIP, convert to a draft 
- [x] When this PR is ready, mark it as "Ready for review"
- [x] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewers

## References
#5263 
_References to pertaining issue(s)_

## QA Steps

1. `git checkout pr_branch_name`
2. `npm install`
3. `npm run dev-reinit`
4. `npm run dev`

List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos.

## Reviewers

_Reviewers for this PR_

### CODE REVIEW

**Comment by barankyle** [comment]
The text file viewing is good. I'd like it to be better formatted, rather than be one big block of text without the empty lines that I know are in many of those files; formatted JSON, with indentation for each sub-level of values, would also be nice. Still don't see the name/filetype/size of selected files/folders.

**Comment by FabriceIRANKUNDA** [comment]
> The text file viewing is good. I'd like it to be better formatted, rather than be one big block of text without the empty lines that I know are in many of those files; formatted JSON, with indentation for each sub-level of values, would also be nice. Still don't see the name/filetype/size of selected files/folders.

Good! To see the properties of a file/folder just do right click on a folder/file and choose properties and you can see that.

File: packages/editor/src/components/assets/FileBrowserGrid.tsx [hunk 1]
```diff
@@ -185,6 +218,7 @@ function FileBrowserItem(props: FileBrowserItemType) {
           <MenuItem onClick={Copy}>{t('editor:layout.filebrowser.copyAsset')}</MenuItem>
           <MenuItem onClick={rename}>{t('editor:layout.filebrowser.renameAsset')}</MenuItem>
           <MenuItem onClick={deleteContentCallback}>{t('editor:layout.assetGrid.deleteAsset')}</MenuItem>
+          <MenuItem onClick={viewAssetProperties}>{t('editor:layout.filebrowser.viewAssetProperties')}</MenuItem>
```
Comment (barankyle) [inline packages/editor/src/components/assets/FileBrowserGrid.tsx:221]: It looks like this needs an entry in i18n/en/editor.json.

**Review by barankyle — CHANGES_REQUESTED** [review]
I would prefer for file names in the list to have the extension on them, e.g. 'xrengine.config.ts' instead of just 'xrengine.config'. To me, it's easier to know what they all are at a glance, rather than having to right click and then open the file properties to see the file type. I see the little icons to the left of the name that indicate model/image/text/etc., but I think seeing the extension would help, too.

I would also like to see a "[breadcrumbs](https://en.wikipedia.org/wiki/Breadcrumb_navigation)" directory hierarchy along the top of the file browser, so that you can see what folder you're currently in. Once you start diving down folders, you sometimes aren't sure where you are; seeing that you're in projects/project_1/assets/ would help, as would the ability to quickly jump back to the root of a project in a single click rather than having to click the back button several times.

**Review by barankyle — APPROVED** [review]

**Comment by HexaField** [comment]
Screenshot from 2022-03-05 13-01-57
folder properties do not display folder path in URL

**Comment by HexaField** [comment]
When trying to delete a folder
```
[server] [server] [Error: EISDIR: illegal operation on a directory, unlink '/home/josh/Desktop/XREngine/packages/server/upload/projects/msa'] {
[server] [server]   errno: -21,
[server] [server]   code: 'EISDIR',
[server] [server]   syscall: 'unlink',
[server] [server]   path: '/home/josh/Desktop/XREngine/packages/server/upload/projects/msa'
[server] [server] }
```

**Comment by HexaField** [comment]
Screenshot from 2022-03-05 13-05-55
model preview should take up whole area

**Comment by HexaField** [comment]
Screenshot from 2022-03-05 13-07-11
model preview should only have white lights (see discolouration)

**Review by HexaField — CHANGES_REQUESTED** [review]
left some review comments

**Pull request merged**